specs/Deployment/DeploymentAPI.md

```diff
@@ -181,8 +198,8 @@ they shut down, to refer to the updated framework package.
 
 #### OnError
 
-Additional behavior is available if an error occurs in `DeploymentManager.Initialize()`,
-per the following logic:
+Additional behavior is available if an error occurs in `DeploymentManager.Initialize()`, per the
+following logic:
 
 ```
 DeploymentManager::Initialize(...)
```

Comment:
I don't understand what I'm reading. Is this app code? If not, why are we putting non-app code in the documentation?

If app code, please make this real code, not some weird blend of Visual Basic, C++, and C#.

What does "Only needed supported in packaged processes" mean? What is the thing that is needed only in packaged processes?

I can't find any function called `IsPackagedProcess()`. Is it part of WinAppSDK? The only hit I can find is [an internal function inside the BCL](https://referencesource.microsoft.com/#System.Xml/parent/parent/parent/parent/InternalApis/NDP_Common/inc/Win8Helpers.cs,84).

Also, the test appears to be backward. Don't you mean `if !IsPackagedProcess()`?

Also, why can't you call Initialize for an unpackaged process? Earlier, we said that unpackaged full trust apps can use the Deployment API to get the WinAppSDK packages installed.

Please don't use jargon like "pop the debugger" or "pop some UI". 

Is `options.OnError_ShowUI` the same as `deploymentInitializationOptions.OnErrorShowUI`? I thought `deploymentInitializationOptions.OnErrorShowUI` was telling the `DeploymentManager.Initialize` method that it should display UI, not a note to ourselves to remember to show UI later.